explainers/signed-exchange-alternate-link-explainer.md

```diff
@@ -0,0 +1,164 @@
+# Signed Exchange alternate link
+## Introduction
+We want to extend the usage of the existing [rel=alternate](https://html.spec.whatwg.org/multipage/links.html#rel-alternate) link header for signed exchange. Using this link header, the content publishers can declare that the resource is available in signed exchange format. This can be used both by the crawlers of aggregator sites  (SNS, News site, search engine..) and by the UAs. The crawlers can cache and serve the signed exchange of the content in their own server. The UAs can provide the users with a way to save the page in signe exchange format. And also signed exchange alternate links can be used to recursively prefetch appropriate subresource signed exchanges while prefetching the main resource signed exchange.
+
+## Use Cases
+### Signed Exchange discovery of main resource
```

Comment:
```suggestion
### Discovery of a signed exchange for the main resource
```

However, I'm not sure this use needs to be in an explainer at all. The "Otherwise" branch in https://html.spec.whatwg.org/multipage/links.html#rel-alternate already says that "The keyword creates a hyperlink referencing an alternate representation of the current document. The nature of the referenced document is given by the hreflang, and type attributes." There's nothing new in saying that a signed exchange can be an alternate representation of a document. So maybe delete this whole sub-section.